pkg/handler/crud_test.go

```diff
@@ -849,20 +849,6 @@ func TestCrudVariantsWithFailures(t *testing.T) {
 		assert.NotZero(t, *res.(*variant.PutVariantDefault).Payload)
 	})
 
-	t.Run("PutVariant - put invalid attachment", func(t *testing.T) {
```

Comment:
This test was removed as it is now allowed to store artbirary JSON structures

Reply:
I think we can just change the test to accept string => int payload.

Reply:
Will do, might also add a test for more complex types